### PR TITLE
Redux setup

### PR DESCRIPTION
**Pull request by joe-afternightfall**

### CODE REVIEW

**Comment by joe-afternightfall** [comment]
sooooo i just noticed i spelled our company name wrong when naming the organization.  

ill wait till after we merge all our work into master, then rename the org and we'll update our remote origins

File: src/configs/redux/store.ts
Comment (dave-afternightfall) [inline src/configs/redux/store.ts:26]: not sure i understand what this is for.  can you help me understand what defaultData is?

**Review by dave-afternightfall — APPROVED** [review]
looks good

File: src/configs/redux/store.ts
Comment (joe-afternightfall) [reply src/configs/redux/store.ts:26]: just dummy data to make sure i had the redux store setup correctly.  ill make sure it gets deleted

**Pull request merged**
